### DISCUSSION TITLE
Set working mode

### DISCUSSION THREAD

**Discussion by timconinx** [wlcrs/huawei_solar#502] [General]
I don't know if this exists or should be a feature, but is there a service to set the working mode of the Luna battery 
```
options:
  - Maximise Self Consumption
  - Time Of Use
  - Fully Fed To Grid
friendly_name: Battery Working Mode
```
I can change the setting through the HA user interface, but the `huawei_solar.` services don't seem to support this... am I missing something?

**Comment by Roving-Ronin** [comment]
@timconinx you need to add this under issues, as a 'feature request'.

https://github.com/wlcrs/huawei_solar/issues/new?created_from_discussion_number=502
